tools/refguide_check.py

```diff
@@ -55,13 +57,23 @@ def get_all_dict(module):
         except ValueError:
             pass
 
+    # FIXME: shouldn't modules be in the refguide, actually? if they're in __all__?
```

Comment:
It's currently not possible to distinguish between public and private submodules based on their names, and the vast majority that will be detected here will be private. The Scipy namespace is quite flat, so not much to detect here: https://github.com/scipy/scipy/blob/master/doc/API.rst.txt